ui.frontend/src/customFunctions.js

```diff
@@ -86,8 +86,36 @@ function navigateTo(destinationURL, destinationType) {
     }
 }
 
+/**
+ * Default Error handler for invoke service rule
+ * @name defaultErrorHandler Default Error Handler Function
+ * @errorHandler
+ */
+function defaultErrorHandler(response, headers, globals) {
```

Comment:
please document wrt response, headers and global object so that a developer can refer to this function to implement a custom handler.

Reply:
Done